octopus_deploy/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,283 @@
+name: Octopus Deploy
+files:
+- name: octopus_deploy.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+    - template: init_config/http
+  - template: instances
+    options:
+    - name: octopus_endpoint
+      display_priority: 8
+      description: |
+        Octopus API endpoint.
+        See https://octopus.com/docs/octopus-rest-api/getting-started#authentication for more details.
+      value:
+        example: http://localhost:80
+        type: string
+      required: true
+    - name: spaces
+      display_priority: 5
+      description: |
+        Optional configuration to indicate the spaces that we want to be processed. If not configured,
+        all spaces and their corresponding project groups and projects will be processed.
+
+        The 'include' key will indicate the regular expressions of the spaces for which metrics are to be reported
+        and the configuration to be applied to each of them. Each group may have a 'project-groups'-like configuration,
+        enabling or disabling metric collection for the project groups which match that condition. For further details
+        see next section 'project-groups'. If no configuration associated with the key is indicated with the regular
+        expression, they will be processed with the default configuration.
```

Comment:
```suggestion
        The 'include' key indicates the regular expressions of the spaces for reporting metrics
        and the configuration applied to each of them. Each group can have a 'project-groups'-like configuration that
        enables or disables metric collection for the project groups matching that condition. For further details,
        see the next section, 'project-groups'. If no configuration associated with the key is indicated with the regular
        expression, they are processed with the default configuration.
```

Reply:
Updated, thanks! I also removed will be from other config documentation